apps/wfo-ui-surf/translations/en-GB.json

```diff
@@ -41,10 +41,34 @@
                         "impactOnSubscriptions": "Impact on subscriptions",
                         "subscriptionImpactTable": {
                             "subscriptionTitle": "Subscriptions",
+                            "subscriptionDescription": "Subscription:",
```

Comment:
If we eventually want dutch translations too, maybe add all theses keys with english content to the dutch file with a `Todo` in front of every text so we don't forget any translations